### DISCUSSION TITLE
Is wildcard Prefix available for trace Ignore plugin for Endpoints

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#11329] [General]
We have gone through that below document but there is nothing prefix  available for trace Ignore plugin for Endpoints of skywalking java agent.I want to remove */management/prometheus and this app name changes based on app team.Please let us know is there any upcoming skywalking java version will support that or it supports in current version.
example:
GET:/tesla/management/prometheus

**Comment by wu-sheng** [comment]
The ignore plugin is there, please. The suffix you mentioned is only for skipping the resource file access. 

This is the real ignore plugin. https://skywalking.apache.org/docs/skywalking-java/v9.0.0/en/setup/service-agent/java-agent/agent-optional-plugins/trace-ignore-plugin/

**Reply by skundu1234** [reply]
Thanks @wu-sheng .Does this ignore plugin support below path like this.

trace.ignore_path=**/your/path/1/, **/your/path/2/

Example provided for this below in document:
trace.ignore_path=/your/path/1/** ,   /your/path/2/**

**Reply by wu-sheng** [reply]
Yes. But you need to pay the costs of performance. Prefix wildcard is slow in matching.

**Reply by skundu1234** [reply]
Thanks for your reply.

**Reply by skundu1234** [reply]
@wu-sheng  is there anything we can ignore in OAP level if we don't want to change anything in agent side.

**Reply by wu-sheng** [reply]
I don't think. The reason is still the same. This costs too much, if we allowed this on the server side, the CPUs of OAP would be burnt out.
